### PR TITLE
hugo v0.15

### PR DESCRIPTION
**Pull request by joneskoo**
go_resources generated with:

  homebrew-go-resources github.com/spf13/hugo

Replaced manually maintained resources with output from script. Seems like a lot of work to do it any other way than by script?

gen autocomplete syntax had changed.

This is my first change to a go formula…

### CODE REVIEW

File: Library/Formula/hugo.rb [hunk 1]
```diff
@@ -186,7 +178,7 @@ def install
 
     system "go", "build", "-o", bin/"hugo", "main.go"
 
-    system bin/"hugo", "genautocomplete", "--completionfile=#{buildpath}/hugo.sh"
+    system bin/"hugo", "gen", "autocomplete", "--completionfile=#{buildpath}/hugo.sh"
```
Comment (dunn) [inline Library/Formula/hugo.rb]: The release notes say `hugo gen` can create manpages too. Could you add a second command for that? Thanks so much!

**Comment by joneskoo** [comment]
@dunn Added man pages. Unfortunately doesn't look like the command supports target dir, but luckily brew is flexible!

https://github.com/spf13/hugo/blob/3d09de891072985e6c59f899f09132a76d44f7cd/commands/genman.go#L19

**Comment by dunn** [comment]
Thanks so much; merged in fbca49bba5e9f478a9dec4e5dd84e8c73e79912d.  If you feel like pestering the developers to make `gen man` more flexible, that'd be awesome too!

**Comment by anthonyfok** [comment]
You are right!  I thought I did add a configurable destination man page output directory in my pull request https://github.com/spf13/hugo/pull/1602, but it turns out that I really didn't!  No wonder nothing shows up with `hugo gen man --help`!  (Oops!)

Thank you for working around that in the Homebrew recipe(?)/script.  I will get that fix in soon to go in for v0.16.

Cheers,
Anthony

**Pull request closed without merging**